src/Web/Extensions/TempDataExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Newtonsoft.Json;
+
+namespace Contoso.FraudProtection.Web.Extensions
+{
+    public static class TempDataExtensions
```

Comment:
TempData sounds fishy; what exactly are these for?

Reply:
This is for showing the API request/response on the following page, one time and one time only.  TempData exists until the next request and then automatically goes discarded.  It specifically helps in scenarios where we redirect from our controller rather than just "post back" where we would otherwise have a harder time persisting this data.